### PR TITLE
Create trip page

### LINKED ISSUE #35: Create Trip Page
Add a create trip modal.

### PR DESCRIPTION
**Pull request by Kanav-Arora**
# Description

Closes #35

### CODE REVIEW

File: frontend/src/components/feature/Add Trip/Day.js
Comment (Aadi71) [inline frontend/src/components/feature/Add Trip/Day.js:1]: Why adding space in the name???
It should be: `AddTrip` or `Add_Trip`

File: frontend/src/components/feature/NavBar/AddTripButton.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { WriteMini } from '../../../assets/ext-icon'
+
+export default function AddTripButton(props) {
+    return (
+        <button onClick={props.onClick} className='text-white pr-2 flex flex-row items-center'>
```
Comment (Aadi71) [inline frontend/src/components/feature/NavBar/AddTripButton.js:7]:
```suggestion
        <button onClick={props.onClick} className='text-white pr-2 flex-row items-center'>
```

**Review by Aadi71 — APPROVED** [review]

**Comment by Aadi71** [comment]
Please attach screenshots for the FE changes u have made.

File: frontend/src/components/feature/NavBar/AddTripButton.js (hunk 1 above)
Comment (Kanav-Arora) [reply frontend/src/components/feature/NavBar/AddTripButton.js:7]: In Tailwind CSS 
Flex is the superclass and flex-row is the class property
Definition of class is mandatory unlike css
PFL

**Pull request merged**